package.json

```diff
@@ -62,23 +64,27 @@
     "@testing-library/react": "^10.3.0",
     "@testing-library/react-hooks": "^3.3.0",
     "@testing-library/user-event": "^12.8.3",
-    "babel-eslint": "^10.0.0",
-    "babel-loader": "^8.0.5",
-    "babel-plugin-styled-components": "^1.10.0",
+    "@types/styled-components": "^5.1.11",
+    "@typescript-eslint/eslint-plugin": "^4.28.4",
+    "@typescript-eslint/parser": "^4.28.4",
+    "babel-eslint": "^10.1.0",
+    "babel-loader": "^8.2.2",
+    "babel-plugin-styled-components": "^1.13.2",
     "css-loader": "^2.1.0",
     "cypress": "^6.3.0",
     "cypress-file-upload": "^5.0.2",
     "cypress-plugin-tab": "^1.0.5",
     "dotenv": "^8.0.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-prettier": "^4.0.0",
-    "eslint-plugin-better-styled-components": "^1.1.1",
-    "eslint-plugin-cypress": "^2.11.2",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jsx-a11y": "^6.1.1",
-    "eslint-plugin-react": "^7.11.1",
-    "eslint-plugin-react-hooks": "^1.6.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb": ">= 18",
+    "eslint-config-prettier": ">= 8",
+    "eslint-plugin-better-styled-components": "^1.1.2",
+    "eslint-plugin-cypress": "^2.11.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": ">= 3",
```

Comment:
Why use `>=` instead of `^`
I think that `^` is safer because 
```
^3.1.4 is the same as  >=3.1.4 <4.0.0
```
whereas  `>=` will allow the package to be updated to the any higher version, including major changes.